spec/unit/screenshoter.unit.spec.js

```diff
@@ -128,14 +128,14 @@ describe("Screenshoter unit", function() {
 
   it("should merge user config", function() {
     screenshoter.config = {
-      screenshotPath: 'REPORTS',
+      screenshotPath: 'REPORTS2',
```

Comment:
`REPORTS` causes an error `Error: EEXIST: file already exists, mkdir '/app/REPORTS'` on Windows. It looks conflict with the existing `reports`. I changed the directory name to avoid the problem though it rather looks like an issue of `mkdirp`.